cypress/e2e/spec.cy.ts

```diff
@@ -0,0 +1,137 @@
+describe('E2E Test', () => {
+  it('Opens the root page', () => {
+    cy.visit('http://localhost:3000');
```

Comment:
Can you look at adding a `beforeEach` block and moving all the `cy.visit` calls into that single block?

Reply:
I've added a beforeEach block and moved the first `cy.visit('http://localhost:3000');` in most test cases, save for the one which needs to pre-register a GET and PUT.